lib/game.js

```diff
@@ -38,9 +38,10 @@ class Game {
     }
     if (scores.length > 0) {
       var sortedScores = scores.sort(function(a, b){return b-a;});
-      $('#score').empty();
+      var $score = $('#score')
+      $score.empty();
```

Comment:
Score is a good use of this! Looks good. 

http://giphy.com/gifs/ship-tDafHUBVrRKtq